test/unit/lexer/location_offset.d

```diff
@@ -135,51 +135,51 @@ unittest
 @("multibyte character token")
 unittest
 {
-    enum code = "🍺";
+    enum code = "'🍺'";
 
     scope lexer = new Lexer("test.d", code.ptr, 0, code.length, 0, 0);
 
     lexer.nextToken;
 
-    assert(lexer.token.loc.offset == 4, code);
+    assert(lexer.token.loc.offset == 0, code);
 }
 
-@("multibyte string token")
+@("multibyte character string token")
 unittest
 {
-    enum code = "🍺🍺";
+    enum code = "\"🍺🍺\"";
```

Comment:
For the outer string literal, I recommend using backticks to avoid needing to escape the double quotes.